docs/customizations/parts/custom-slots-example.md

```
@@ -1,21 +1,51 @@
-<script setup lang="ts">
-async function confetti() {
-  (await import('https://esm.sh/canvas-confetti')).default()
-}
-</script>
+<OAOperation operationId="createArtist">
 
-<OAOperation operationId="getAllArtists">
+<template #header="header">
+<SlotDebugger :slotProps="header" title="header" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: SlotDebugger

(MD033, no-inline-html)